src/js/components/award/shared/awardAmounts/AwardAmountsChart.jsx

```diff
@@ -682,10 +684,13 @@ const AwardAmountsChart = ({
                 return (
                     <HorizontalSingleStackedBarViz {...chartPropsOutlays} />
                 );
+            } else if (isDirectPayment || isGrant || isOther) {
+                return (
+                    <HorizontalSingleStackedBarViz {...chartPropsOutlays} />
+                );
             }
-
             return (
-                <RectanglePercentViz {...chartProps} />
```

Comment:
We don't want to switch this chart type over to the HorizontalBarViz yet without testing.  There's a separate ticket to do this work.

Reply:
whoops okay i'll fix this